portal-frontend/src/app/features/edit-submission/parcel-details/parcel-entry/parcel-entry.component.html

```diff
@@ -145,12 +145,12 @@ <h5>Parcel Lookup</h5>
       <mat-button-toggle-group class="input" id="isFarm" formControlName="isFarm">
         <mat-button-toggle
           value="true"
-          [ngClass]="{ 'error-outline': isFarm.invalid && (isFarm.dirty || isFarm.touched) }"
+          [ngClass]="{ 'error-outline': parcelForm.touched || (isFarm.invalid && (isFarm.dirty || isFarm.touched)) }"
```

Comment:
This will immediately show an error outline once any form field is touched is it expected?

Reply:
Fixed!